docs/user_guide/HOWTO/Configure_Ssh.rst

```diff
@@ -1,118 +1,95 @@
+for the OS and environment. To override this and **disable** management of one
+or both of these classes, add the following to :term:`Hiera`:
 
-.. NOTE::
+.. code-block:: yaml
 
-   The examples below feature ``include 'ssh::server'`` and ``include
-   'ssh::client'``, but most SIMP scenarios already include them both via
-   ``ssh``.  So, for SIMP systems, you will customize parameter settings
-   for ``ssh::server`` and ``ssh::client`` via Hiera.
+   ssh::enable_client: false
+   ssh::enable_server: false
 
 
 Managing Settings for the SSH Server
 ------------------------------------
 
-Including ``ssh::server`` with the default options will manage the server with
-reasonable settings for each host's environment.
+As stated above, the ``ssh::server`` class is included by default by the
+``simp-ssh`` module and configured with "sane" settings for each host's
+environment. Detailed descriptions of the various settings are provided in the
+sshd_config(5) man page.
 
 
 Configuring ``ssh::server::conf`` from Hiera
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To customize the SSH server, edit the parameters of ``ssh::server::conf`` using
-:term:`Hiera` or :term:`ENC`.
+Many standard sshd_config settings for the SSH server are included with default
+values in the ``ssh::server::conf`` class. To customize the SSH server, edit
+the parameters of ``ssh::server::conf`` using Hiera or :term:`ENC`.
 
 .. NOTE::
 
    Unlike many SIMP modules, these customizations cannot be made
    directly with a resource-style class declaration―they *must* be
    made via automatic parameter lookup provided by Hiera or ENC.
+   Examples using Hiera are provided for illustrative purposes.
 
 In Hiera:
 
 .. code-block:: yaml
 
    ssh::server::conf::port: 2222
-   ssh::server::conf::ciphers:
-   - 'chacha20-poly1305@openssh.com'
-   - 'aes256-ctr'
-   - 'aes256-gcm@openssh.com
-
-In Puppet:
-
-.. code-block:: puppet
-
-   include 'ssh::server'
-
-   # Alternative:
-   # if `ssh::enable_server: true`, this will also work
-   include 'ssh'
+   ssh::server::conf::passwordauthentication: false
 
 
 Managing Additional Settings with ``sshd_config``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To manage SSH server settings that are not managed by the SIMP ``ssh`` module,
-use the ``sshd_config`` resource from `augeasproviders_ssh`_.  This is what the
-SIMP ``ssh`` module uses internally to manage the ``/etc/ssh/sshd_config``
-file, and you can use it to set any options ``ssh::server::conf`` does not
-manage.
-
-For instance, to set the sshd ``LogLevel`` option to ``VERBOSE``:
-
-.. code-block:: puppet
+To manage global sshd_config settings that are not included in the
+``ssh::server::conf`` class, specify them in Hiera with the
+``ssh::server::conf::custom_entries`` parameter as follows:
 
-   # VERBOSE will log SSH key fingerprints used for logins
-   sshd_config { 'LogLevel' : value => 'VERBOSE' }
-
-
-Mixing ``ssh::server::conf`` and ``sshd_config``
```

Comment:
Was there a reason to remove this example instead of updating it?   It was added to address user feedback in ([SIMP-4197](https://simp-project.atlassian.net/browse/SIMP-4197)).

Reply:
LogLevel is managed by pupmod-simp-ssh now.  So, that example would result in a duplicate resource and fail.